### DISCUSSION TITLE
French channel disapears

### DISCUSSION THREAD

**Discussion by martynojunk** [iptv-org/community#590] [Q&A] [answered]
hello, 
there is a few frenc channels disapearing those last weeks 
TF1
france 2 
france 3 
france 4
france 5
TF! series
Tmc
Arte
W9
tfx
Is it possible to update please ?
Thank you so much

**Comment by rphlmm** [comment]
culprit: @Dum4G
https://github.com/iptv-org/iptv/commit/7588693c1ceef822a96c352146dff7f9984a7c4c 
He removed all French channels, without explanation.

**Reply by chili-debug** [reply]
hi, do you know if this will/can be fixed? Alternatively, do you where or how I can watch France 2 outside of France?

**Reply by Dum4G** [reply]
What explanation do you need for dead source and February commit?

**Reply by chili-debug** [reply]
Sorry, I am not familiar with coding language, I was just looking to watch TV. After research I found the channel I was looking for else where. Have a good day.

**Comment by martynojunk** [comment]
Hello, what solution did you find please? I have the same issue. Thank you

Le jeu. 25 juil. 2024 à 20:46, chili-debug ***@***.***> a
écrit :

**Reply by chili-debug** [reply]
Hi, I found a .m3u file on the internet. This file contains the link you can copy in VLC for example to watch the channel.

**Reply by martynojunk** [reply, marked as answer]
is it possible to have a link for m3u file please?
thank you

**Reply by Timelost2** [reply]
Hello, I would also be very grateful if you could share that link please

**Reply by chili-debug** [reply]
Hi, it depends on the channel you are looking for. I found the .m3u file on https://alliptvlinks.com/latest-channels/   It is free and I didn't have any problem with it. Search for the channel you want. You can then download the file (you might have to disable your ad blocker). I had to try multiple files until I got on that worked fine. If like me you are searching for France 2, I found that "FR RP FRANCE 2 HD Backup NO_1" worked best. Hope this is helpful.

**Reply by Nic787** [reply]
I was actually looking about how to watch the French channels. Thank you for providing an alternative.

**Reply by rphlmm** [reply]
@chili-debug @martynojunk @Timelost2 @Nic787 
Added some up to date French channels:

#EXTINF:-1 tvg-id="Arte",Arte (720p)
https://artesimulcast.akamaized.net/hls/live/2031003/artelive_fr/index.m3u8
#EXTINF:-1 tvg-id="Eurosport 1",Eurosport 1 (1080p)
http://145.239.5.177:8080/529/index.m3u8
#EXTINF:-1 tvg-id="Eurosport 2",Eurosport 2 (720p)
http://ipro.webvps.de:25443/live/o1tEXUfCO0/QE4I5CeVb1/9280.m3u8
#EXTINF:-1 tvg-id="France 2", France 2 (720p)
http://tv.pro-ott.com:25443/live/HAJG6/k7bKPqg05y/15226.m3u8
#EXTINF:-1 tvg-id="France 3", France 3 (540p)
http://ipro.webvps.de:25443/live/o1tEXUfCO0/QE4I5CeVb1/271306.m3u8
#EXTINF:-1 tvg-id="France 5", France 5 (720p)
http://tv.pro-ott.com:25443/live/HAJG6/k7bKPqg05y/15167.m3u8
#EXTINF:-1 tvg-id="France TV Info", France TV Info (1080p)
http://ipro.webvps.de:25443/live/o1tEXUfCO0/QE4I5CeVb1/43444.m3u8

**Comment by martynojunk** [comment]
That's great.  Thank you so muchEnvoyé depuis mon appareil Galaxy
-------- Message d'origine --------De : Raphaël Martin Melizi ***@***.***> Date : 31/07/2024  17:55  (GMT+00:00) À : "iptv-org/iptv-org.github.io" ***@***.***> Cc : martynojunk ***@***.***>, Mention ***@***.***> Objet : Re: [iptv-org/iptv-org.github.io] French channel disapears (Discussion iptv-org/iptv-org.github.io#1761) 
@chili-debug @martynojunk @Timelost2 @Nic787
Added some up to date French channels:
#EXTINF:-1 tvg-id="Arte",Arte (720p)
https://artesimulcast.akamaized.net/hls/live/2031003/artelive_fr/index.m3u8
#EXTINF:-1 tvg-id="Eurosport 1",Eurosport 1 (1080p)
http://145.239.5.177:8080/529/index.m3u8
#EXTINF:-1 tvg-id="Eurosport 2",Eurosport 2 (720p)
http://ipro.webvps.de:25443/live/o1tEXUfCO0/QE4I5CeVb1/9280.m3u8
#EXTINF:-1 tvg-id="France 2", France 2 (720p)
http://tv.pro-ott.com:25443/live/HAJG6/k7bKPqg05y/15226.m3u8
#EXTINF:-1 tvg-id="France 3", France 3 (540p)
http://ipro.webvps.de:25443/live/o1tEXUfCO0/QE4I5CeVb1/271306.m3u8
#EXTINF:-1 tvg-id="France 5", France 5 (720p)
http://tv.pro-ott.com:25443/live/HAJG6/k7bKPqg05y/15167.m3u8
#EXTINF:-1 tvg-id="France TV Info", France TV Info (1080p)
http://ipro.webvps.de:25443/live/o1tEXUfCO0/QE4I5CeVb1/43444.m3u8

**Comment by Nic787** [comment]
For some reasons none of them work on my TV. They are all geo-blocked. I don't really want to use a VPN on my TV, so I guess I will try to find an alternative for that.
